### DISCUSSION TITLE
Broken dependency errors with Dialogic 2.0 Alpha 6

### DISCUSSION THREAD

**Discussion by nlupugla** [dialogic-godot/dialogic#1468] [Q&A]
Hi Everyone,

I recently tried to upgrade my dialogic 2.0 version from alpha 5 to alpha 6 in a game I'm making in Godot 4.0 with some visual novel elements. When I loaded up my project, I got several broken dependency errors. For example, something (possibly game_ui.tscn ?) in the project is looking for res://addons/dialogic/Events/Text/node_input_handler.gd, but can't find it. Any ideas on how I should resolve this? Has anyone else encountered these broken dependencies or is it something to do with how I upgraded dialogic versions?

Looking through the alpha 5 to alpha 6 changelog, I noticed that node_input_handler.gd was renamed to default_input_handler.gd. Should I fix the dependency to point to default_input_handler.gd instead? Should I do something similar for the other dependency errors I'm getting?

Thanks in advance!

**Comment by Jowan-Spooner** [comment]
Yes this is a known issue that should be fixed. Thanks for the report! The default input handler shouldn't be in any scene, as it is automatically instanced at runtime.

**Reply by Jowan-Spooner** [reply]
I think these errors are fixed on main now.
